quickstart.py

```diff
@@ -261,14 +296,10 @@ def serversResponding(config):
       print "Timeout exceeded waiting for SymServer to start"
       return False
 
-    # Make sure we have a port value to contact SymServer on.
-    if 'configPath' in config:
-      SymServer.config.loadFile(config['configPath'])
-    elif 'configJSON' in config:
-      SymServer.config.loadJSON(config['configJSON'])
+    port = getSymServerConfig(config)['port']
```

Comment:
Perhaps loading configuration from environment variables is desirable, but I think that should be filed as an issue rather than part of this pull request. It is not really related to Docker.